packages/@azure/servicebus/data-plane/test/stress/fixedNumberOfClientsMultipleQueues.ts

```diff
@@ -0,0 +1,62 @@
+import { ServiceBusClient, SendableMessageInfo } from "../../lib";
+
+const connectionString = "";
+
+const testDurationInMilliseconds = 60000 * 5 * 12 * 24 * 7; // 1 week
+
+const numOfClients = 10;
+
+let msgId = 1;
+
+let snapshotIntervalID: any;
+
+let isJobDone = false;
+
+async function main(): Promise<void> {
+  snapshotIntervalID = setInterval(snapshot, 5000); // Every 5 seconds
+  setTimeout(() => {
+    isJobDone = true;
+  }, testDurationInMilliseconds);
+
+  sendReceiveMessages();
```

Comment:
We should add an `await` here.
If we don't then any error coming from inside `sendReceiveMessages()` will result in `UnhandledPromiseRejectionWarning`